addon/models/draft-registration.js

```diff
@@ -0,0 +1,19 @@
+import DS from 'ember-data';
+import OsfModel from './osf-model';
+
+export default OsfModel.extend({
```

Comment:
Huzzah, new models!

We've recently started adding documentation comments to our model files, so that implementers can easily find information on what each field means (and how to use the model). Could you do the same for these new models?

These links typically point at production swagger docs, with a direct link to the section for that endpoint. (an intern is making those links work better on the swagger end). See examples:
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/models/file-version.js#L6